docs/tiltfile_concepts.md

```diff
@@ -12,6 +12,8 @@ Functions like `k8s_yaml` and `docker_build` register information. At the end of
 
 Because your Tiltfile is a program, you can configure it with familiar constructs like loops, functions, arrays, etc. This makes Tilt more extensible than a configuration that requires hard-coding all possible options up-front.
 
+Any relative paths in your Tiltfile are evaluated relative to the location of the Tiltfile. (Absolute paths are also fine!)
```

Comment:
Should we encourage people to use absolute paths?

Reply:
I don't think so -- seems like it adds more mental overhead, makes it harder to just scan a tiltfile and get what's happening, etc.

(Unless you're saying I should remove the last because it implicitly encourages abs paths and we DON'T want that?

Reply:
Yeah that question is more what I'm asking :)

Absolute paths are annoying in shared code because someone else might have that file in a different path.

Reply:
Eh i don't have strong feelings. I can remove that parenthetical.